test/benchmarks/sails.load.test.js

```diff
@@ -205,10 +206,6 @@ function setupBenchmarks() {
  * @this {Array} benchmarks
  */
 function reportBenchmarks() {
-  var _ = require('lodash');
```

Comment:
btw re: lodash isregexp- you're totally right-- we're heading the direction of using the individual lodash packages to help mitigate future versioning concerns.  That's a work in progress, but we'd welcome any help you could offer there in a separate PR (i.e. changing `var _ = require('lodash');` to `var isRegExp = require('lodash.isregexp');` etc)